x-pack/docs/en/security/authentication/internal-users.asciidoc

```diff
@@ -2,8 +2,10 @@
 [[internal-users]]
 === Internal users
 
-The {stack-security-features} use four _internal_ users (`_system`, `_xpack`,
-`_xpack_security`, and `_async_search`), which are responsible for the operations
+NOTE: These users are designed for internal use by {es}. Direct use is not supported.
```

Comment:
```suggestion
NOTE: These users are designed for internal use by {es} only. Authenticating with these users is not supported.
```

Reply:
I think that including "only" strengthens the wording here. I also attempted to define what "direct use" means and provide more context.